src/software/ai/hl/stp/tactic/simulated_penalty_goalie_tactic_test.cpp

```diff
@@ -0,0 +1,52 @@
+#include <gtest/gtest.h>
+
+#include <utility>
+
+#include "software/ai/hl/stp/tactic/penalty_goalie_tactic.h"
+#include "software/geom/algorithms/contains.h"
+#include "software/simulated_tests/simulated_tactic_test_fixture.h"
+#include "software/simulated_tests/terminating_validation_functions/ball_kicked_validation.h"
+#include "software/simulated_tests/terminating_validation_functions/robot_state_validation.h"
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/test_util/test_util.h"
+#include "software/time/duration.h"
+#include "software/world/world.h"
+
+class SimulatedPenaltyGoalieTacticTest : public SimulatedTacticTestFixture
+{
+};
+
+TEST_F(SimulatedPenaltyGoalieTacticTest, test_initial_position_on_goal_line) {
```

Comment:
You could set an initial velocity on the ball and check that is blocks it

Reply:
Added this but I need some help setting with my tactic constructor